function/pom.xml

```diff
@@ -19,13 +19,23 @@
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-starter-function-webflux</artifactId>
+			<artifactId>spring-cloud-starter-function-web</artifactId>
```

Comment:
I think there may be a bug in SCF when using Webflux and `Function<CloudEvent, CloudEvent>` (unrelated to WASM because I pulled it out of the equation when debugging it). SCF failed during output message conversion due to missing `contentType` header (which is available when using WebMvc). I will follow  up on this w/ Oleg and switch this back to WebFlux once I figure it out. For now, I went back to WebMvc in the spirit of forward progress.